app/code/Magento/CatalogGraphQl/Model/Resolver/Products/Query/Search.php

```diff
@@ -84,6 +91,7 @@ public function __construct(
         $this->fieldSelection = $fieldSelection;
         $this->productsProvider = $productsProvider;
         $this->searchCriteriaBuilder = $searchCriteriaBuilder;
+        $this->resolveCategoryAggregation = $resolveCategoryAggregation;
```

Comment:
Hello @korovitskyi 
These changes are backward incompatible.
Could you please fix it.